board/safety/safety_subaru.h

```diff
@@ -20,6 +20,14 @@ const SteeringLimits SUBARU_GEN2_STEERING_LIMITS = {
   .type = TorqueDriverLimited,
 };
 
+const LongitudinalLimits SUBARU_LONG_LIMITS = {
+  .min_gas = 0,
+  .max_gas = 3400,
```

Comment:
Make sure to label how much acceleration in m/s^2 this applies

Reply:
looks like it changes with speed, you can do something similar to GM like in https://github.com/commaai/openpilot/pull/25453 when you start to tune. Doesn't look like it can engine brake which is interesting. Maybe RPM can do that.

Reply:
We usually allow 2 m/s in the safety, you can bump it